src/nidmm/system_tests/test_system_nidmm.py

```diff
@@ -52,7 +52,7 @@ def test_wrong_parameter_type(device_info):
 
 def test_warning(device_info):
     with nidmm.Session(device_info['name']) as session:
-        session.configure_measurement_digits(nidmm.Function.RES_2_WIRE, 1e6, 3.5)
+        session.configure_measurement_digits(nidmm.Function._2_WIRE_RESISTANCE, 1e6, 3.5)
```

Comment:
Weird to use _ because this enum value happens to start with number.
DAQmx spells it out when needed so would be: TWO_WIRE_RESISTANCE

I don't think it's for this PR but I will open an issue about it.